src/components/encounter/ohri-encounter-form.component.tsx

```diff
@@ -494,6 +497,20 @@ export const OHRIEncounterForm: React.FC<OHRIEncounterFormProps> = ({
       };
     }
 
+    if (encounterForSubmission.form.uuid === '1e5614d6-5306-11e6-beb8-9e71128cae77') {
```

Comment:
Form Engine should not know about the `form uuid` It is possible for another implementation to have a different Uuid for the same form